app/models/news/collection.rb

```diff
@@ -19,5 +19,15 @@ def id
     def to_param
       slug.presence || id
     end
+
+    def matches_param?(collection_id)
```

Comment:
Isn't this something backend can do?

Reply:
At present, no

I _could_ add a `#show` action to do it _but_ I'm already requesting _all_ collections from the backend so adding another api request (and supporting API on the backend) seems a pretty complicated solution to 'find this item in the array I already have')

Will go just iterating the list I already have for now